src/patternfly/components/FormControl/form-control.scss

```diff
@@ -149,4 +149,12 @@ $pf-c-form-control__select--ViewBox: "320" !default;
       --pf-c-form-control--invalid--Background: var(--pf-c-form-control__select--invalid--Background);
     }
   }
+
+  &.pf-m-resize-vertical {
+    resize: vertical;
+  }
+
+  &.pf-m-resize-horizontal {
+    resize: horizontal;
+  }
```

Comment:
Can you add `max-width: 100%` and a `min-width` to this?

Reply:
@mceledonia should we add a `min-width` to the textarea when you can resize it horizontally, to keep a user from sizing it down too small?